docs/user-guide/getting-started.md

```diff
@@ -30,6 +30,8 @@ $ npx eslint --init
 $ yarn run eslint --init
 ```
 
+**Note:** `--init` assumes you have a `package.json` file already. If you don't, make sure to run `npm init` or `yarn create` beforehand.
```

Comment:
Should this be `yarn init` instead of `yarn create`?

Reply:
Oops, yes

Reply:
```suggestion
**Note:** `--init` assumes you have a `package.json` file already. If you don't, make sure to run `npm init` or `yarn init` beforehand.
```